### DISCUSSION TITLE
Videos, I want videos!

### DISCUSSION THREAD

**Discussion by richardhovdsveen** [dailydotdev/daily#1020] [Feature Requests]
Like, short good explanation videos of different technologies and methodologies, ted-talks and smart innovations. Inspiring.

**Comment by idoshamun** [comment]
Hi @richardhovdsveen,

We're actively working on adding YouTube support to our content pipeline and UI.
Stay tuned!

**Comment by idoshamun** [comment]
YouTube support is live for quite some time now :)
